### ISSUE TITLE
[bug] Docs say syntax one way, but output is another way.

### ISSUE THREAD

**Issue by trusktr** [cssinjs/jss#845]
__Expected behavior:__

The docs on [Alternative syntax for space and comma separated values](http://cssinjs.org/json-api?v=v9.8.7#alternative-syntax-for-space-and-comma-separated-values) say that

```js
export default {
  button: {
      border: ['1px solid red', '1px solid blue'],
  },
};
```

should compile to

```css
.button-0-1-179 {
  border: 1px solid red, 1px solid blue;
}
```

__Describe the bug:__

But instead it is missing a comma, so the output looks like

```css
.button-0-1-179 {
  border: 1px solid red 1px solid blue;
}
```

so there's either a bug with the docs, or a bug with the lib. Which one?

Note that the docs for [jss-extended Full syntax](http://cssinjs.org/jss-expand-full/?v=v5.3.0#using-arrays-for-space-separated-properties) conflict with the Core JSON syntax doc.

The output for

```js
  a: {
    border: [
      // Numbers can become default unit automatically.
      [1, 'solid', 'red'],
      [1, 'solid', 'blue']
    ]
  },
```

looks as expected, with a comma.

__Codesandbox link:__ 

Paste

```js
export default {
  button: {
      border: ['1px solid red', '1px solid blue'],
  },
  a: {
    border: [
      // Numbers can become default unit automatically.
      [1, 'solid', 'red'],
      [1, 'solid', 'blue']
    ]
  },
};
```

into the [repl](http://cssinjs.org/repl).

(P.S. It would be neat if the repl updated the URL with a query so we can paste a repl URL and the visitor can see the same code, like the Babel repl).

__Versions (please complete the following information):__
 - see repl

**Comment by kof** [comment]
Yeah, mb jss-expand should not do this for border.

**Comment by kof** [comment]
I assume its not a bug its a defect, since probably its what we designed jss-expand to do. Core without 
 plugin will do what docs says.

**Comment by kof** [comment]
Mb though it needs to be fixed in the docs, because for jss-expand it probably does make sense. Its just confusing that we have different output for the same syntax.

Plugins should add syntax, but not change the existing syntax in a confusing way.

**Comment by HenriBeck** [comment]
Yeah, I would agree that plugins should only add syntax and not change the core syntax. Either we remove the syntax from the core or the plugin aligns itself with the core.

**Comment by kof** [comment]
We discussed another potential way of doing it: if we look at the overall project, most people will use it with the default preset.

What if we format documentation as if default preset IS the syntax, and only mention in a separate markdown file the syntax differences without the plugins. 

Previously core was the main base and plugins an optional thing. In this case, with plugins it would be the baseline, naked core would be an optional thing, for those who know what they are doing.

**Comment by HenriBeck** [comment]
I think formatting the documentation with the default syntax the one from the preset is the wrong idea.
I think we should drop the syntax from the core and just have the `jss-expand` syntax.

**Comment by kof** [comment]
which part of the syntax in the core would you drop?

**Comment by kof** [comment]
there are basically 2 things with arrays: [] and [[]]

**Comment by HenriBeck** [comment]
I would drop in generall arrays in the core and just move the array syntax in the `jss-plugin-syntax-expand` plugin.

**Comment by kof** [comment]
can be done, yeah

**Comment by kof** [comment]
Btw. even in the core, the simple [] syntax can be used, because it needs nothing special, it will be just converted to a string by js.
